server/implementation/src/main/java/io/smallrye/graphql/spi/MetricsService.java

```diff
@@ -6,4 +6,8 @@ public interface MetricsService {
     void start(Long measurementId, Context context);
 
     void end(Long measurementId);
+
+    void up(Context context);
```

Comment:
Could we make it more clear what this is for? If it's for subscriptions only, I'd name these methods `subscriptionStart` and `subscriptionEnd`

Reply:
@jmartisk changes applied